src/main/java/emu/grasscutter/game/combine/CombineResult.java

```diff
@@ -4,11 +4,11 @@
 import java.util.List;
 import lombok.Getter;
 import lombok.Setter;
-
+@Setter @Getter
```

Comment:
```suggestion

@Setter @Getter
```
this is my bad for not adding the extra line lol